### PR TITLE
Update introduction

### PR DESCRIPTION
**Pull request by aphelionz**
This PR essentially re-works the entire introduction, in the attempt to make it as concise and relevant as possible.

### CODE REVIEW

File: src/introduction.md [hunk 1]
```diff
@@ -1,42 +1,39 @@
 # Introduction
 
-[Substrate] is a blockchain and smart contract framework written in the Rust programming language.
-The InterPlanetary File System (or [IPFS]) is a distributed, peer-to-peer,
-content-addressed storage framework.
+`offchain::ipfs` is [Substrate], infused with [IPFS].
```
Comment (ljedrz) [inline src/introduction.md:3]:
```suggestion
`offchain::ipfs` is [Substrate] infused with [IPFS].
```

File: src/introduction.md [hunk 2]
```diff
@@ -1,42 +1,39 @@
...
 
-Substrate has many features that differentiate it from a typical blockchain framework. One
-such feature is [Off-Chain Workers], which are separate wasm execution contexts that run alongside
-your blockchain node’s runtime, enabling you to make send and receive data from external sources,
-such as HTTP APIs.
+By including [Rust IPFS](https://github.com/rs-ipfs/rust-ifps) in the native
+Substrate runtime, and then by allowing pass-through wasm calls via Substrate's
```
Comment (ljedrz) [inline src/introduction.md]:
```suggestion
Substrate runtime, and by allowing pass-through wasm calls via Substrate's
```

File: src/introduction.md (hunk 1 above)
Comment (aphelionz) [reply src/introduction.md:3]: I like the comma :) I'm gonna keep it for now

**Comment by aphelionz** [comment]
@whalelephant Anything else to add? Otherwise @ljedrz do you approve?

**Pull request merged**